mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -75,6 +75,7 @@ var (
 		"addons_config.0.network_policy_config",
 		"addons_config.0.cloudrun_config",
 		"addons_config.0.gcp_filestore_csi_driver_config",
+		"addons_config.0.gcs_fuse_csi_driver_config",
```

Comment:
should we move only `gcs_fuse_csi_driver_config ` to 'ga' block or should we move everything in `addonsConfigKeys` ?

I understand that `istio_config` and `kalm_config` are deprecated. What is the effect of moving `gcs_fuse_csi_driver_config` to 'ga' block ?

Reply:
Just `gcs_fuse_csi_driver_config` should move into the block. Typically in cases like these, I just move them into the existing blocks that contain `istio_config` and `kalm_config`, thus they are all grouped together and its easier to read IMO. It shouldn't matter that the others are deprecated, they're still in the code and will be handled in the same manner wrt the provider. Does that make sense?

Essentially the effect is that the new config block won't make it to the `google` provider, but only the `google-beta` provider. You can look at [this comment](https://github.com/GoogleCloudPlatform/magic-modules/pull/7884#issuecomment-1538875019) above and it will show what the code will look like once they're generated downstream.

For example, you'll see that `istio_config` is in the [`google-beta` provider](https://github.com/search?q=repo%3Ahashicorp%2Fterraform-provider-google-beta%20istio_config&type=code), but it doesn't exist in the [`google` provider](https://github.com/search?q=repo%3Ahashicorp%2Fterraform-provider-google%20istio_config&type=code).  (Except for docs, we standardize on beta docs)

Reply:
Thanks for the information.

I've updated it in the latest iteration.

Reply:
thanks! and sorry for the mis-understanding, but you'll want to move all the code below that references it into blocks as well. if you do a search in the code for `istio_config` you'll see what i mean, i think.